js/models/communityresource.js

```diff
@@ -11,7 +11,9 @@ export default class CommunityResource extends Model {
     fetch(ident) {
         ident = ident || this.id;
         if (ident) {
-            this.$api('datasets.retrieve_community_resource', {community_resource: ident}, this.on_fetched);
+            this.$api('datasets.retrieve_community_resource', {
```

Comment:
Note for later: I think we should review the naming patterns to have the same all accross the application.